docs/src/en/reference/server/plugin.md

```diff
@@ -0,0 +1,89 @@
+---
+title: plugin system
+icon: plugin
+---
+
+Users can extend the custom hook function through the Hook provided by Waline to realize custom functions. But if users want to share custom Hook methods, they can only use the copy method. To solve this problem, the Waline plugin system came into being.
+
+## Install the plugin
+
+A new `plugins` attribute has been added to Waline's initial configuration, which supports configuring multiple plugins.
+
+```js
+// index.js
+const Waline = require('@waline/vercel');
+const HelloWorldPlugin = require('@waline-plugins/hello-world');
+
+
+module.exports = Waline({
+   plugins: [
+     Hello World Plugin,
+   ],
+});
+```
+
+In addition to directly installing other people's plug-ins, we can also configure our own plug-in logic here.
```

Comment:
To install others' plugin directly, you can also place plugin hooks direct in `plugins`: